docs/index.js

```diff
@@ -20,24 +20,35 @@ app.post("/feedback/submit", async (req, res) => {
   });
 
   const feedback = req.body
+  const today = new Date().toISOString().slice(0, 19)
+
+  // Validate URL in feedback
+  if ((typeof feedback.url !== 'string') || !feedback.url.startsWith(config.getPrimaryRoute().url)) {
+    return res.status(400).send("The submitted URL isn't valid")
+  }
+
+  // Validate feedback itself
+  if ((typeof feedback.feedback !== 'string') || !['positive', 'negative'].includes(feedback.feedback)) {
+    return res.status(400).send("The submitted feedback isn't valid")
+  }
 
   // Create a feedback table if it doesn't exist
   try {
     await connection.query(
       `CREATE TABLE IF NOT EXISTS Feedback (
```

Comment:
Unrelated to the security issues, I would suspect a SELECT here would be much more performant than a `create if exists`. 

(this assumes the schema/database is named `main`. Change as needed)
e.g. 
`select count(TABLE_NAME) from information_schema.tables where table_schema = 'main' and table_name = 'feedback';` 

and then check the result to make sure it is 1.